### ISSUE TITLE
Homework #1 available

### ISSUE THREAD

**Issue by ategge** [tegge-classroom/STAT2984-2018#1]
Homework #1 is available in the Assignments folder. Please follow the link for submission: https://classroom.github.com/a/noUPBTFA.

**Comment by maxh95** [comment]
Just to verify, our homework is to fill out the Assignment 1: Introductions and then submit to GitHub?

**Comment by ategge** [comment]
Correct. Fill out the blanks in Assignment 1, and submit to GitHub using the link above.

**Comment by ategge** [comment]
Reminder: Assignment 1 is due tomorrow morning before class. 

If you have any questions or are having trouble submitting the assignment, please start an issue on GitHub (or email me). If you had success submitting the assignment, please help out your fellow classmates resolve their problems. If all else fails, email me your assignment. 

Good luck!

**Comment by jthough7** [comment]
Hello this is Jonathan Hough, attached I have my homework assignment, I was going to try to explain my problem to you but I feel like it would be easier to in class.

**Comment by apappy56** [comment]
Hi my Name is Aaron Pappy and i was struggling

**Comment by apappy56** [comment]
Hi my name is Aaron Pappy and i was struggling to send my homework so i am
just going to attach it to this email. Sorry for the previous message.

Fill in the following three blanks.

Name: Aaron Pappy Major: Statistics Expected graduation date: Spring 2021

Answer the following four questions.

Why are you taking this class? Because it was recommened to me by Dr. Jane
Robertson Evia and i also want to possibly pursue a computer science minor
with my Statistics major so this class will help me diversify myself and
pick up more skill in working with computers What is your previous
programming experience? I took intro to computational thinking which was
taught by Dr. Kafura What do you want to take away from this class? I want
to be able to increase my skills with working with computers Do you have
any apprehensions about this class? If so, describe. I know this class may
end up being a little confusing at times but i am excited to see how this
class improves my skills with computers.

**Comment by ategge** [comment]
FYI, if you reply to an 'issue alert' from your email, it will post your response as a reply to the issue on GitHub :-)

**Comment by chloeliu97** [comment]
Assignment 1: Introductions
Link: https://classroom.github.com/a/noUPBTFA
Due: Tuesday, January 23, 2018 @ 8:00am

Fill in the following three blanks.

Name:Xiaoran Liu
Major:BIT
Expected graduation date:2020


Answer the following four questions.

Why are you taking this class?
This class is related to things I want to study in graduate school.

What is your previous programming experience?
I have taken CS 1054 at Tech during my freshman year, but I was not good at coding. 

What do you want to take away from this class?
Be more familiar with coding. 

Do you have any apprehensions about this class? If so, describe.
I am afraid that I will not be able to learn to code fast.
